test/lib/mayaUsd/fileio/testPrimWriter.py

```diff
@@ -49,6 +49,10 @@ def __init__(self, *args, **kwargs):
         self._SetUsdPrim(usdPrim)
         primWriterTest.InitCalled = True
 
+    @classmethod
+    def CanExport(cls, exportArgs, exportObj=None):
+        return mayaUsdLib.PrimWriter.ContextSupport.Supported
```

Comment:
I think adding a flag "CanExportCalled" like in the other test would help verify the functionality.